static/css/cf-enhancements.less

```diff
@@ -500,3 +500,18 @@ tbody {
     name: EOF
     eof: true
 */
+
+/**
+ * This is a temporary hack to fix the contact expandables. CF Expandables could use a
+ * larger rework, especially removing duplicated blocks (open/close states, double search
+ * filter buttons based on device size, etc)
+ */
+.expandable__padded .expandable_header {
+  padding-right: 90px;
```

Comment:
Ahhhh, right. I mistakenly assumed you were adding the padding to the `expandable_label`.